src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -811,6 +814,12 @@ def rsa_padding_supported(self, padding: AsymmetricPadding) -> bool:
         else:
             return False
 
+    def rsa_encryption_supported(self, padding: AsymmetricPadding) -> bool:
+        if self._fips_enabled and isinstance(padding, PKCS1v15):
+            return False
+        else:
+            return self.rsa_padding_supported(padding)
```

Comment:
This will return true for PSS, do we care?

Reply:
I think I'm fine with that for now